### ISSUE TITLE
Other participant sees stripes on the screen I share (Screensharing issue)

### ISSUE THREAD

**Issue by milaholanda** [jitsi/jitsi-meet#13966]
### Description:
I'm a teacher and I have some students who, whenever I share my screen, see only stripes on a distorted image. [Like this](https://photos.app.goo.gl/fZpR53M7n2UjjujZ9). It's not a constant problem and it only happens to two students, so I get the feeling it's something related to how their computers deal with jitsi, because on other platforms we don't have any problem screensharing.

A funny thing that happened today was that I was screensharing on Chrome and there were stripes, I tried on Edge, stripes too, but for the short moment in which I was in call on both browsers, the student reported the image to be visible. 

### Steps to reproduce:

1. Start the call as usual. I use either Edge version 118.0.2088.46 or Chrome 118.0.5993.88 
2. Hit the screensharing button and pick the tab you'd like to share.
3. Check if the other person sees a good image.

### Expected behavior:

A visible image should be seen by other participants.

### Actual behavior:

One of the other participants sees a striped screen and distorted images.  [See student's picture here](https://photos.app.goo.gl/fZpR53M7n2UjjujZ9)

### Server information:

- Jitsi Meet version: not sure but probably the latest cause the school has just implemented it
- Operating System:

### Client information:

- Browser / app version: Edge version 118.0.2088.46 or Chrome 118.0.5993.88
- Operating System: Windows 10

**Comment by damencho** [comment]
Is this on meet.jit.si or your own deployment? If your deployment what are the jitsi-meet and videobridge versions?

**Comment by jdsatava127** [comment]
We also ran into this issue on our instance. We had a meeting where 1 user could not see the screen share. It was just squiggly lines. The Chrome version is 121.0.6167.185. I've attached some screenshots of the issue

Jitsi version works - 2.0.9258 - docker tag stable-9258
Jitsi version broken - 2.0.9364 - docker tag stable-9364

![Clipboard - April 2, 2024 10_11 AM (2)](https://github.com/jitsi/jitsi-meet/assets/66796767/6b7424be-222d-4a45-9560-ff19318d6e32)

![Clipboard - April 2, 2024 10_10 AM](https://github.com/jitsi/jitsi-meet/assets/66796767/28bb71dd-2c76-4276-894a-72e956ea5a69)
